packages/elements/src/components/ui/banner/Banner.stories.tsx

```diff
@@ -81,12 +86,27 @@ export const LoadingNoHeader = () => (
 
 export const BottomContent = () => (
   <Box width={"500px"}>
-    <Banner
-      variant={"error"}
-      headerText={"This is header"}
-      text={"This is not working."}
-    >
-      <PrimaryButton label={"Retry"} />
+    <Banner variant={"error"} headerText={"This is header"}>
+      <ul style={{ paddingInlineStart: "20px" }}>
```

Comment:
No custom styling should be required to make it look good.

Reply:
removed list, it should be implemented as a new component